packages/firestore/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
   `FirestoreSettings` to `true`. Note that the current behavior
   (`DocumentSnapshot`s returning JS Date objects) will be removed in a future
   release. `Timestamp` supports higher precision than JS Date.
+- [feature] Added ability to control whether DocumentReference.get() and
+  Query.get() should fetch from server only, cache only, or attempt server and
+  fall back to the cache (which was the only option previously, and is now the
+  default.)
```

Comment:
super nit: period should go after paren, I think?

Reply:
Done.